samples/SqlServerCacheConcurencySample/Program.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.Memory;
+using Microsoft.Framework.Caching.SqlServer;
+using Microsoft.Framework.Logging;
+
+namespace SqlServerCacheConcurencySample
+{
+    /// <summary>
+    /// This sample requires setting up a Microsoft SQL Server based cache database.
+    /// 1. Create a new database or use as existing gone.
+    /// 2. Run the command "dnx . install-sqlservercache <connectionstring-here> <name-of-table-to-be-created>"
+    ///    to setup the table.
+    /// </summary>
+    public class Program
+    {
+        private const string Key = "MyKey";
+        private static readonly Random Random = new Random();
+        private DistributedCacheEntryOptions _cacheEntryOptions;
+
+        public void Main()
+        {
+            _cacheEntryOptions = new DistributedCacheEntryOptions();
+            _cacheEntryOptions.SetSlidingExpiration(TimeSpan.FromSeconds(10));
+
+            var loggerFactory = new LoggerFactory();
+            loggerFactory.AddConsole();
+            var cache = new SqlServerCache(new SqlServerCacheOptions()
+            {
+                ConnectionString = "Server=localhost;Database=ASPNET5CacheTest;Trusted_Connection=True;",
```

Comment:
I reccomend removing references to AspNet5. This is a Ms.Framework component, not an AspNet component.